src/resources/ha-date-picker-style.html

```diff
@@ -0,0 +1,93 @@
+<dom-module id="ha-date-picker-vaadin-date-picker-styles" theme-for="vaadin-date-picker">
+  <template>
+    <style>
+      [part~="clear-button"] {
```

Comment:
This disables the clear button on all vaadin-date-pickers. That's not good.

Reply:
True. I first targeted this to the history panel. Then realized those style template apply to all pickers once loaded, which is fine of course, except for hiding the clear button indeed.
I think I can make that depend on a ‘required’ attribute.